telco-examples/edge-clusters/dhcp-less/edge-metallb-multi-node/telco-capi-metallb-multi-node.yaml

```diff
@@ -74,90 +74,68 @@ spec:
                 ExecStartPost=/bin/sh -c "umount /mnt"
                 [Install]
                 WantedBy=multi-user.target
-            - name: metallb-endpointcopier-operators.service
-              enabled: true
-              contents: |
-                [Unit]
-                Description=metallb-endpointcopier-operators
-                Wants=network-online.target
-                After=rke2-server.service
-                [Service]
-                Type=oneshot
-                User=root
-                ExecStartPre=/bin/sh -c "while [ ! -d /var/lib/rancher/rke2/server/manifests ]; do sleep 1; done"
-                ExecStart=/bin/sh -c "cp /var/metallb-endpointcopier-operators.yaml /var/lib/rancher/rke2/server/manifests/metallb-endpointcopier-operators.yaml"
-                [Install]
-                WantedBy=multi-user.target
-            - name: metallb-cr.service
-              enabled: true
-              contents: |
-                [Unit]
-                Description=metallb-cr
-                Wants=network-online.target
-                After=metallb-endpointcopier-operators.service
-                [Service]
-                Type=oneshot
-                User=root
-                ExecStartPre=/bin/sh -c "while [ ! -f /var/lib/rancher/rke2/bin/kubectl ] && [ ! -f /etc/rancher/rke2/rke2.yaml ]; do sleep 1; done"
-                ExecStart=/bin/sh -c "while [ $(/var/lib/rancher/rke2/bin/kubectl --kubeconfig=/etc/rancher/rke2/rke2.yaml get deployment --ignore-not-found metallb-controller | wc -l) -eq 0 ]; do sleep 1; done"
-                ExecStart=/bin/sh -c "/var/lib/rancher/rke2/bin/kubectl --kubeconfig=/etc/rancher/rke2/rke2.yaml rollout status deployment metallb-controller -n default --timeout=150s"
-                ExecStart=/bin/sh -c "cp /var/metallb-cr.yaml /var/lib/rancher/rke2/server/manifests/"
-                ExecStartPost=/bin/sh -c "cp /var/endpoint-svc.yaml /var/lib/rancher/rke2/server/manifests/"
-                [Install]
-                WantedBy=multi-user.target
         storage:
           files:
```

Comment:
I'd say you better add a folder creation here as well:

```
        storage:
          directories:
            - path: /var/lib/rancher/rke2/server/manifests
              overwrite: true
```

Reply:
but the rke2-capi-provider has to be created it before (the ignition is executed after the postCommands...) so the folder should be always present before starting this part. 
BTW I've tested on the lab and it worked as expected:
```
localhost:/var/lib/rancher/rke2/server/manifests # ls
endpoint-copier-operator.yaml  metallb.yaml	  rke2-ingress-nginx.yaml	     rke2-snapshot-controller.yaml
endpoint-svc.yaml	       rke2-canal.yaml	  rke2-metrics-server.yaml	     rke2-snapshot-validation-webhook.yaml
metallb-cr.yaml		       rke2-coredns.yaml  rke2-snapshot-controller-crd.yaml
```

Reply:
Ok, I mean, it won't hurt if present but if it is not needed that's fine.